mobile-app/app/screens/AppNavigator/screens/Dex/DexScreen.tsx

```diff
@@ -336,6 +336,7 @@ function TopLiquiditySection ({ pairs, onPress }: DexScrollableSectionProps): JS
           style={tailwind('mr-2')}
           onPress={onPress}
           label={translate('screens/DexScreen', 'Swap')}
+          testId='Composite_Swap'
```

Comment:
if you do this, all your swap button will have the same test id, how do you differentiate them?